docs/src/PageHeader.doc.js

```diff
@@ -0,0 +1,643 @@
+// @flow strict
+import React, { type Node } from 'react';
+import { PageHeader } from 'gestalt';
+import PropTable from './components/PropTable.js';
+import DocsPageHeader from './components/PageHeader.js';
+import MainSection from './components/MainSection.js';
+
+const cards: Array<Node> = [];
+const card = (c) => cards.push(c);
+
+card(
+  <DocsPageHeader
+    name="Page Header"
+    description="PageHeader is used to indicate the title of the current page, as well as optional actions."
+    shadedCodeExample
+    defaultCode={`
+      function IntroPageHeaderExample() {
+        const [open, setOpen] = React.useState(false);
+        const [selected, setSelected] = React.useState([]);
+        const anchorRef = React.useRef(null);
+        const handleSelect = ({ item }) => {
+          if (selected.some((selectedItem) => selectedItem.value === item.value)) {
+            setSelected((selected) =>
+              selected.filter((selectedItem) => selectedItem.value != item.value),
+            );
+          } else {
+            setSelected((selected) => [...selected, item]);
+          }
+        };
+
+        return (
+          <PageHeader
+            title="Product groups"
+            subtext="S. E. All products USD"
```

Comment:
Could we add a subtext that is easier to understand as metadata? `S. E. All products USD` I have no idea what this means haha

Reply:
In this case it's the name of the Product Group. I think this is a real example from business, @ashleyseto is there something else we could put here?